Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Projection/ProjectionIntegrityViolationDetector.php

```diff
@@ -656,7 +662,7 @@ protected function findProjectedContentStreamIds(): iterable
     protected function findProjectedDimensionSpacePoints(): DimensionSpacePointSet
     {
         $records = $this->client->getConnection()->executeQuery(
-            'SELECT DISTINCT dimensionspacepoint FROM ' . $this->tableNamePrefix . '_hierarchyrelation'
+            'SELECT dimensionspacepoint FROM ' . $this->tableNamePrefix . '_dimensionspacepoints'
         )->fetchAllAssociative();
```

Comment:
I wonder if we couldn't use the new DimensionSpacePoints repository here?

Reply:
I guess, I undesrtood this here as really low level facility bypassing anything else to make sure only the database is checked?